.github/header-checker-lint.yml

```diff
@@ -4,6 +4,8 @@ ignoreFiles:
   - receiver/**/generated_metrics.go
   - receiver/**/generated_metrics_test.go
   - receiver/mongodbreceiver/**/*.go
+  - receiver/prometheusreceiver/**/*.go
```

Comment:
Doesn't this pattern cover both or nah?

Reply:
I think you're right. Just tested it, god I had globstar